.gitignore

```diff
@@ -1,3 +1,12 @@
 *.o
 *.a
-*.dblite
\ No newline at end of file
+*.dblite
+# Matlab ASV autosave files
+*.asv
+# Pychache files 
```

Comment:
Would you mind deleting the trailing whitespace here.

Reply:
done (i think)